cli/tests/unit/process_test.ts

```diff
@@ -476,19 +474,27 @@ unitTest(
       cmd: [Deno.execPath(), "eval", "setTimeout(() => {}, 10000)"],
     });
 
-    Deno.kill(p.pid, "SIGINT");
-    const status = await p.status();
-
-    assertEquals(status.success, false);
-    try {
-      assertEquals(status.signal, "SIGINT");
-    } catch {
-      // TODO(nayeemrmn): On Windows sometimes the following values are given
-      // instead. Investigate and remove this catch when fixed.
-      assertEquals(status.code, 130);
-      assertEquals(status.signal, 2);
+    if (Deno.build.os === "windows") {
+      // currently not implemented
+      assertThrows(() => {
+        Deno.kill(p.pid, "SIGINT");
+      }, Error);
+      p.close();
```

Comment:
I'd suggest to move `p.close()` back to the end of the function, rather than duplicating it in the `if{}` and `else{}` blocks.